examples/python/spi.py

```diff
@@ -22,21 +22,24 @@
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE
 
-import mraa as m
-import random as rand
+# Example Usage: Read from MCP3004 ADC pin 0 in single ended mode
 
-# Excuse the super boring example, I was out of fun devices to play with, this
-# will write and read the same data back to itself, a few 100 times, just short
-# MISO & MOSI on your board
+import mraa
+import time
 
-dev = m.Spi(0)
+# initialise SPI
+dev = mraa.Spi(0)
 
-for x in range(0,100):
-  txbuf = bytearray(4)
-  for y in range(0,4):
-    txbuf[y] = rand.randrange(0, 256)
-  rxbuf = dev.write(txbuf)
-  if rxbuf != txbuf:
-    print("We have an error captain!")
-    exit(1)
+# prepare data to send
+txbuf = bytearray(3)
+txbuf[0] = 0x01
+txbuf[1] = 0x80
+txbuf[2] = 0x00
 
+while True:
+    # send data through SPI
+    rxbuf = dev.write(txbuf)
+    value = ((rxbuf[1] & 0x03) << 8) | rxbuf[2]
+    print value
```

Comment:
Ok, this was one of the two conflicts between my fixes and your first version of the PR. You've picked your version here, but this reverts my fix - and will produces Sonar hits. Should be using `print()` form, like all other places we use this function.